### PR TITLE
Improve CI

### PR DESCRIPTION
**Pull request by henrysun007**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change (select or add applied and delete the others)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] API change with a documentation update
- [ ] Additional test coverage
- [X] Code cleanup or just sync with upstream third-party crates

## Checklist

- [X] Fork the repo and create your branch from `master`.
- [X] If you've added code that should be tested, add tests.
- [X] If you've changed APIs, update the documentation.
- [X] Ensure the tests pass (see CI results).
- [X] Make sure your code lints/format.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -193,8 +193,8 @@ if(USE_PREBUILT_MESAPY)
 add_custom_command(
   OUTPUT ${MESAPY_OUTPUTS}
   COMMAND
-    wget -qN https://mesapy.org/release/${MESAPY_VERSION}-mesapy-sgx.tar.gz &&
-    tar xzf ${MESAPY_VERSION}-mesapy-sgx.tar.gz
+    wget -qN https://github.com/mesatee/mesapy/releases/download/${MESAPY_VERSION}/mesapy-${MESAPY_VERSION}.tar.gz &&
```
Comment (bigdata-memory) [inline CMakeLists.txt:196]: I tried to open this URL, but got 404. Please double check it, thanks.

Comment (henrysun007) [reply CMakeLists.txt:196]: Can you post your URL here? The URL here is https://github.com/mesatee/mesapy/releases/download/v1.0.0-sgx/mesapy-v1.0.0-sgx.tar.gz after `MESAPY_VERSION` is filled.

Comment (bigdata-memory) [reply CMakeLists.txt:196]: @mssun will help to find a link for this PR, thanks.

**Review by mssun — APPROVED** [review]
Thanks! Sorry for the late reply. I was not feeling well in recent months. Now, I'm back to the community.

The PR looks good to me.

**Pull request merged**